lib/matplotlib/collections.py

```diff
@@ -1394,29 +1433,25 @@ def __init__(self, segments,     # Can be None.
             `~.path.Path.CLOSEPOLY`.
 
         **kwargs
-            Forwareded to `.Collection`.
+            Forwarded to `.Collection`.
         """
-        if colors is None:
-            colors = mpl.rcParams['lines.color']
-        if linewidths is None:
-            linewidths = (mpl.rcParams['lines.linewidth'],)
-        if antialiaseds is None:
-            antialiaseds = (mpl.rcParams['lines.antialiased'],)
```

Comment:
The `Collection` default is `patch.antialised`, so it looks like this can't be removed.

Reply:
Good catch.  I fixed it by adding _get_default_antialiased methods to Collection and LineCollection for consistency.